dockerfiles/release.dockerfile

```diff
@@ -0,0 +1,33 @@
+# This dockerfile is thought to create a release so you don't need to install every dependeny on your host.
+
+FROM opensuse/leap:15.2
+
+WORKDIR /build
+
+ENV container docker
+
+# Update the repositories and update any system packages
+RUN ["zypper", "-n", "update"]
+# Install webserver and system tools needed for Cobbler
+RUN ["zypper", "-n", "in", "apache2", "apache2-devel", "acl", "apache2-mod_wsgi-python3", "ipmitool", "rsync"]
+RUN ["zypper", "-n", "in", "fence-agents", "genders", "xorriso", "tftp", "hardlink"]
+# Install Python dependencies for Cobbler
+RUN ["zypper", "-n", "in", "python3", "python3-devel", "python3-pip", "python3-setuptools", "python3-future"]
+RUN ["zypper", "-n", "in", "python3-distro", "python3-ldap", "python3-netaddr", "python3-Cheetah3"]
+RUN ["zypper", "-n", "in", "python3-simplejson", "python3-dnspython", "python3-Django", "python3-PyYAML"]
+# Install non packaged dependency
+RUN ["pip3", "install", "pykickstart"]
```

Comment:
We could just... _package_ this. I made a package for pykickstart for other reasons on openSUSE, but nobody had previously expressed any interest, so it wasn't pushed into Factory and Leap 15.2...

Reply:
Alternatively, we could use Fedora. 😉

Reply:
Yes I know. But it is for now not there so we need it like this... Otherwise we would need to wait until this is in Leap 15.2.

Reply:
This image is already pointing to Leap 15.2?

Reply:
Line 3

Reply:
`pykickstart` SRs:
* Factory: https://build.opensuse.org/request/show/807724
* Leap 15.2: https://build.opensuse.org/request/show/807725

Reply:
Fixed through 340ef76